Content.Shared/Eye/VisibilityFlags.cs

```diff
@@ -9,5 +9,6 @@ public enum VisibilityFlags : int
         None   = 0,
         Normal = 1 << 0,
         Ghost  = 1 << 1,
+        AdminObserver = 1 << 2
```

Comment:
Adding another vismask is extremely expensive on engine and I don't think this is worth it to make aghosts see each other.

Reply:
How is it expensive on the engine? I don't know much about that and I am genuinely curious.

Reply:
It's going to add a significant overhead to PVS and limit playercount on the server.